rescript.json

```diff
@@ -27,7 +27,7 @@
   "warnings": {
     "error": "+8"
   },
-  "suffix": ".mjs",
+  "suffix": ".bs.mjs",
```

Comment:
why do we reintroduce the bs.mjs extension?

Reply:
To distinguish the files generated by the compiler from the others, since we are using ESM

Reply:
Is there a technical reason why we need to do that? I intentionally made it that way back then so we can get rid of any extra differentiation.

If we need different extensions, I'd like to not use the bs prefix.

Reply:
Ok, removed